modules/apps/foundation/frontend-theme/space-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,255 @@
+/* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
+
+/* inject:imports */
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+/*
+    ------------------------
+    ----- 1. Variables -----
+    ------------------------
```

Comment:
CSS comments should use a standard format like: ```/* ---------- General ---------- */```.

Reply:
We do not need a CSS comment pointing out variables since we can tell they are variables just based on how they are laid out.